e2etest/pkg/status/bgpstatus.go

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier:Apache-2.0
+
+package status
+
+import (
+	"context"
+	"fmt"
+
+	"go.universe.tf/e2etest/pkg/metallb"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"go.universe.tf/metallb/api/v1beta1"
+	v1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func ForBGPServiceAndNode(cs client.Client, svc *v1.Service, node string) (*v1beta1.ServiceBGPStatus, error) {
```

Comment:
name is a bit confusing,  you mean at the end GetServiceBGPStatusForServiceAndNode, right?

Maybe `status.BGPStatusForServiceAndNode` and a small description?

Reply:
changed to `BGPForServiceAndNode`, is that better? I don't like having status in the name as it's the package's name and feels redundant